Sources/KituraMarkdown/KituraMarkdown.swift

```diff
@@ -34,36 +34,46 @@ public class KituraMarkdown: TemplateEngine {
     /// Create a `KituraMarkdown` instance
     public init() {}
 
-    /// Take a template file in Markdown format and generate HTML format content to 
+
+    /// Take a template file in Markdown format and generate HTML format content to
     /// be sent back to the client.
     ///
     /// - Parameter filePath: The path of the template file in Markdown format to use
     ///                      when generating the content.
     /// - Parameter context: A set of variables in the form of a Dictionary of
     ///                     Key/Value pairs. **Note:** This parameter is ignored at
     ///                     this time
+    /// - Parameter options: useTemplate - The HTML page template to insert rendered markdown into.
+    ///                     Passing in "default" will insert markdown as a child of <body>
+    ///                     Custom HTML template should indicate location of insertion with
+    ///                     <snippetInsertLocation></snippetInsertLocation> tag.
     ///
     /// - Returns: If an Error isn't thrown whenreading the template, a String containing
     ///            an HTML representation of the text marked up using Markdown.
     public func render(filePath: String, context: [String: Any]) throws -> String {
+        return try render(filePath: filePath, context: context, options: [String: Any]())
```

Comment:
No need to create to render methods, a default parameter should be provided instead:
`filePath: String, context: [String: Any], options: [String: Any] = [:]`